assessment.md

```diff
@@ -239,17 +239,39 @@ TODO: add description for each assessment level for this skill.
 
 * Interaction with the team leaders
 
-TODO: add description for each assessment level for this skill.
+- **None**: refuses to interact with and/or work with team leader(s). Ignores advisement or assignment from team leader.
+- **Inadequate**: Interacts with and occasionally works with the team leader(s), but does not actively participate in completing tasks assigned by the team leader(s).
+- **Adequate**: Interacts with and works with the team leader(s) while completing most of the tasks assigned to them by the team leader(s).
+- **Good**: Interacts with, works with, and actively collaborates with the team leader(s). Completes all tasks assigned to them by the team leader(s).
+- **Excellent**: Interacts with, works with, and actively collaborates with the team leader(s). Completes all tasks assigned to them by the team leader(s) and offers additional help to expedite the completion of other related tasks to complete the project.
 
 * Interaction with the course instructor
 
-TODO: add description for each assessment level for this skill.
+- Effective interaction with the course instructor would be being respectful. The both student and instructor should respect each other's time and effort. Students should have appropriate specific questions about programming/ technical problems. Students have to let the instructor know if he/she is understanding the topics and keeping up with her.
+
+- Ineffective interaction with the course instructor would be not coming to office hours and communicating to the instructor. If the students do not tell the what is happening to their study, the teacher can not help the students. Without the instructor's guidance, students will not know if they are going on the right path or not. 
 
 * Interaction with the student technical leaders
 
-TODO: add description for each assessment level for this skill.
+- None: Students don't ask or raise their own concern with TLs at any point of the class. Students also don't response to any question of TL on the chat or on Zoom.
+
+- Inadequate: Students ask or raise their concern no more than 3 times during the class, lab, and practicals. These students seldomly response to TLs' question.
+
+- Adequate: These students ask or raise their concern occasionally during the class. They also corporate with TLs on difficult task. They response to all matters that TLs ask them.
+
+- Good: These students frequently voice out their concerns during the class with TLs. They also use Slack to contact with TLs. They also response to all matters that TLs ask them.
+
+- Excellent: These students regularly voice out their concerns during the class with TLs. They also use Slack to contact with TLs. They also response to all matters that TLs ask them. They also come to TLs' office hours to discuss their problem with the assignments.
```

Comment:
I think that these 3 points should be rephrased to avoid repetition of `They also`. They should also be a little distinct from each other instead of repeating the exact same wording.